book_source/developers_guide/Using-Git.Rmd

```diff
@@ -192,103 +156,52 @@ For PEcAn:
         git branch -D <branchname>
 
 
-
 ### Link commits to issues
 
 You can reference and close issues from comments, pull requests, and commit messages. This should be done when you commit code that is related to or will close/fix an existing issue. 
 
 There are two ways to do this. One easy way is to include the following text in your commit message:
 
 * [**Github**](https://github.com/blog/1386-closing-issues-via-commit-messages)
- * to close: "closes gh-xxx" (or syn. close, closed, fixes, fix, fixed)  
- * to reference: just the issue number (e.g. "gh-xxx")
- * avoid "closes #xxx" which will cross-reference Redmine issues 
+  * to close: "closes gh-xxx" (or syn. close, closed, fixes, fix, fixed)  
+  * to reference: just the issue number (e.g. "gh-xxx")
+  * avoid "closes #xxx" which will cross-reference Redmine issues 
 * **Redmine**: 
- * to close: "fixes redmine #xxx" (or closes etc.) 
- * to reference: "redmine #xxx"
+  * to close: "fixes redmine #xxx" (or closes etc.) 
+  * to reference: "redmine #xxx"
 * **Bitbucket**: 
- * to close: reference and use web interface!
- * to reference: "re #xxx"
+  * to close: reference and use web interface!
+  * to reference: "re #xxx"
```

Comment:
Can we think about deprecating the Redmine cross references -- I haven't seen anyone reference or close an issue over there for years. Also, what's the Bitbucket instructions about? AFAIK there's no parts of PEcAn on Bitbucket to cross-reference.